app/assets/javascripts/sufia/tabs.js

```diff
@@ -1,22 +1,30 @@
+// This code is to implement the tabs on the home page
+
+// navigate to the selected tab or the first tab
+function tabNavigation(e) {
+    $('.nav-tabs').addClass('blah');
```

Comment:
Yeah blah...  I had made the change locally to remove that, but it did not get pushed...